achievements/achievements.py

```diff
@@ -513,7 +513,10 @@ async def award_achievement(ctx, data, name):
         string += f" and earned {','.join(f'{v} {k}' for k, v in achievements[name]['db_rewards'].items())}"
     channel = ctx.author if "hidden" in achievements[name] and achievements[name]["hidden"] else ctx
     try:
-        await channel.send(f"Congratulations {ctx.author.mention}, you just achieved `{name}`{string}!")
+        if name == "Old Man":
+            await channel.send(f"Congratulations {ctx.author.mention}, you just achieved `Old Person`!")
+        else:
+            await channel.send(f"Congratulations {ctx.author.mention}, you just achieved `{name}`{string}!")
```

Comment:
You already fixed the problem by renaming the achievement, this section is pointless.